### PR TITLE
Update supported offboard mode setpoints for rover

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
This PR updates the documentation in rover to include the recent added support for offboard setpoints.
- Attitude setpoints: https://github.com/PX4/Firmware/pull/13323
- Velocity setpoints: https://github.com/PX4/Firmware/pull/13363 https://github.com/PX4/Firmware/pull/13225

### CODE REVIEW

File: en/flight_modes/offboard.md [hunk 1]
```diff
@@ -77,13 +77,18 @@ See https://github.com/PX4/Firmware/pull/12149 and https://github.com/PX4/Firmwa
...
     * Position setpoint (only `x`, `y`, `z`)
+    * Velocity setpoint (only `vx`, `yy`, `vz`)
   - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_LOCAL_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_NED) and [MAV_FRAME_BODY_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_BODY_NED).
 
 * [SET_POSITION_TARGET_GLOBAL_INT](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_GLOBAL_INT)
   * The following input combinations are supported: <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
     * Position setpoint (only `lat_int`, `lon_int`, `alt`)
   - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_GLOBAL](https://mavlink.io/en/messages/common.html#MAV_FRAME_GLOBAL).
 
+* [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET)
```
Comment (hamishwillee) [inline en/flight_modes/offboard.md]: Just FMI, what does it mean to support an attitude setpoint on a rover? Ie You set yaw, but isn't the rest of your attitude  just going to follow ground? Or are we assuming vehicles that can perhaps have the ability to change axle positions etc?

**Comment by hamishwillee** [comment]
Looks good. Is this going into PX4v1.10? If it is then I can merge, if not need to think about whether it is worth holding off until that releases.

File: en/flight_modes/offboard.md (hunk 1 above)
Comment (Jaeyoung-Lim) [reply en/flight_modes/offboard.md]: It supports the attitude messages, but only extracts yaw from the message!

**Comment by Jaeyoung-Lim** [comment]
@hamishwillee I am not sure actually.

File: en/flight_modes/offboard.md (hunk 1 above)
Comment (hamishwillee) [reply en/flight_modes/offboard.md]: Thanks @Jaeyoung-Lim . I added a note for this (please check)

**Comment by hamishwillee** [comment]
OK, well lets hold off submitting then .... because I can't see this inhttps://github.com/PX4/Firmware/commits/release/1.10

**Pull request merged**